addons/binding/org.openhab.binding.netatmo/build.properties

```diff
@@ -13,5 +13,5 @@ bin.includes = META-INF/,\
                lib/commons-codec-1.8.jar,\
                lib/gson-2.3.1.jar,\
                about.html,\
-               lib/netatmo-java-retrofit-1.0.8.jar
```

Comment:
It states that the (binary) file netatmo-java-retrofit-1.0.9.jar was added but not that the other one was removed, so if merged we end up with two files as can be seen in your repo:

https://github.com/clinique/openhab2-addons/tree/Issue2379/addons/binding/org.openhab.binding.netatmo/lib